ui/desktop/app/routes/scopes.js

```diff
@@ -23,9 +44,16 @@ export default class ScopesRoute extends Route {
    * @param {string} params.scope_id
    * @return {Promise{[ScopeModel]}}
    */
-  model() {
+  async model() {
     // NOTE:  In the absence of a `scope_id` query parameter, this endpoint is
     // expected to default to the global scope, thus returning org scopes.
-    return this.store.query('scope', {}).catch(() => A([]));
+    let scopes = A([]);
```

Comment:
Does this need to be an ember array still?

Reply:
I have reverted these changes.